README.md

```diff
@@ -63,7 +63,7 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/a
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+This section has moved here...: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
```

Comment:
- [ ] Please include necessary sections in the readme, such as project title, project description, and getting started instructions.

- [ ] In addition, also deploy the project online and include the live deploy link in the readme.